### ISSUE TITLE
Question-Not-A-Problem: nodejs server only?

### ISSUE THREAD

**Issue by hlubovac** [swagger-api/swagger-editor#1158]
I was hoping the site is a set of static files, so that it could run on arbitrary http server. I followed the instructions (npm install, npm start), and I do see it running on node (although with the same client-JS error repeated hundreds of times; OS windows; see below). I was hoping for the standard `dist` folder, but I don't see any: it seems that the `/dist` part of the path is handled by the server, with no physical files matching the url-path. I'm still hoping that there's something that I missed and that what I want will be possible? While I love nodejs, it's not the environment I can impose where this would be going. Thanks.

JS error unrelated to the question:
`error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style`

**Comment by hlubovac** [comment]
Still unrelated to the question, but that JS error that I reported is actually originating from the server (I see the same printed out in the server console). It's passed onto the client, so when I reported it it was because I saw it in the browser console first.

Sourced from multiple files (paths taken from the console): `./index.js', `./scripts/app.js`, etc (likely all). Might be something related to Windows version of npm and how it handled creating these files during `npm install` process (just my best guess). Sharing this info just in case you care about that particular problem :)
